packages/sdk/src/index.ts

```diff
@@ -45,6 +50,37 @@ export function createRaribleSdk(
 	}
 }
 
+/**
+ * Create middleware controller & wrap methods
+ */
+function setupMiddleware(
+	apis: IApisSdk,
+	internalSdk: IRaribleInternalSdk,
+	sdkContext: ISdkContext
+) {
+	const middlewarer = new Middlewarer()
+
+	middlewarer.use(getInternalLoggerMiddleware(
```

Comment:
We should add configuration from outside to disable logging. By default it must be enabled but user may disable

Reply:
@evgenynacu what you think?